### ISSUE TITLE
GNOME Core Apps List

### ISSUE THREAD

**Issue by OriginalFannibal** [Kazhnuz/awesome-gnome#27]
Found this today: https://blogs.gnome.org/mcatanzaro/2017/08/13/gnome-3-26-core-applications/

Should help us in avoiding ambiguity regarding which apps are classified as core GNOME apps and which aren't.

Here is the entire list as it is right now:  

- Archive Manager (File Roller)
- Boxes
- Calculator
- Calendar (gnome-calendar, not california)
- Characters (gnome-characters, not gucharmap)
- Cheese
- Clocks
- Contacts
- Disk Usage Analyzer (Baobab)
- Disks (gnome-disk-utility)
- Document Viewer (Evince)
- Documents
- Files (Nautilus)
- Fonts  (gnome-font-viewer)
- Help (Yelp)
- Image Viewer (Eye of GNOME)
- Logs (gnome-logs, not gnome-system-log)
- Maps
- Music
- Photos
- Screenshot
- Software
- Simple Scan
- System Monitor
- Terminal
- Text Editor (gedit)
- To Do
- Videos (Totem)
- Weather
- Web (Epiphany)

**Comment by Kazhnuz** [comment]
Nice ! It'll provide good guidelines to mark Core application.

**Comment by Kazhnuz** [comment]
I'm working on the "gnome-core" branch on adding references to this list. I'm trying to find if there is a place where they auto-update the list.

**Comment by OriginalFannibal** [comment]
Seeing how that document refers to the previous update, which too was a similar blog post accompanying the release of GNOME 3.22, I think there isn't a separate auto-updating list that.

What we can do is revisit this issue every 6 months (each GNOME release) and see if a new list has been posted. We could mention the changes here and then add any new apps to the contribution guidelines.

**Comment by Kazhnuz** [comment]
Yep, I didn't find one. I'll try to add in the branch all the core apps in the list this week-end. Should we add in parenthesis the "development" name like "Files (Nautilus)" ?

**Comment by lokesh-krishna** [comment]
Hmmm...  
I think it would be okay to do that. What do you think?

**Comment by Kazhnuz** [comment]
I just feared a bit that it could make the list a bit cluttered, but we have longer names, so it might be fine.
